CHANGELOG.md

```diff
@@ -17,6 +17,10 @@ Non-backwards compatible changes
   significantly faster. However, its reduction behaviour on open terms may have
   changed.
 
+* The definitions of `Algebra.Structures.IsHeyting*` and
```

Comment:
I know this is a draft, but in the final version it would be good to explain broadly what the refactorings actually are here?